std_msgs/Header header
detector/BoundingBox2D[] boundingboxes
